src/utils.ts

```diff
@@ -74,21 +74,14 @@ export function makeTmpDir() {
 }
 
 export function toWinDirFormat(dir: string = ''): string {
-  const results = /\/mnt\/([a-z])\//.exec(dir);
-  if (!results) {
-    return dir;
-  }
-
-  const driveLetter = results[1];
-  return dir.replace(`/mnt/${driveLetter}/`, `${driveLetter.toUpperCase()}:\\`)
-      .replace(/\//g, '\\');
+  return execFileSync('wslpath', [dir]).toString().trim();
```

Comment:
based on the code in https://github.com/mojadev/wsl-path and https://github.com/Microsoft/WSL/issues/2715 don't we need to pass `-w` to go from a WSL path to a windows path which is what this function did prior to this PR?

maybe it's worth renaming this because in chrome-launcher this is used to (sometimes) go the opposite direction

Reply:
No because you're running in a Linux env, so you want the Linux path, but you're right about the two functions.

This PR will most likely completely break Windows support - sorry about that.

Reply:
Ok after looking back over it, I think the main thing is that the names should be improved.

This function is only used by the WSL logic, so is doing what we want while not breaking anywhere else, but the current names could lead people to assume it converts *to* Windows paths, rather than the otherway around.